lib/addons/s3-csi-driver/index.ts

```diff
@@ -16,9 +16,9 @@ const S3_DRIVER_POLICY = 's3-csi-driver-policy';
  */
 export interface S3CSIDriverAddOnProps extends HelmAddOnUserProps {
     /**
-     * ARN of the S3 bucket to be used by the driver
+     * The names of the S3 buckets to be used by the driver
      */
-    s3BucketName: string;
+    s3BucketNames: string[];
```

Comment:
Minor: since it is an s3-csi-driver, s3 is redundant in the names. If we are changing the names, let's make it just `bucketNames`.